sim_derive/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "sim_derive"
+version = "0.7.1"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+syn = "*"
+quote = "*"
```

Comment:
I haven't seen this dependency version format before.  Is this preferred over the usual semantic versioning constraints?

Reply:
I specified required versions for `sync` and `quote` dependencies.